network/transport/v2/conversation.go

```diff
@@ -0,0 +1,201 @@
+/*
+ * Nuts node
+ * Copyright (C) 2022 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package v2
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/nuts-foundation/nuts-node/crypto/hash"
+)
+
+var maxValidity = 30 * time.Second
```

Comment:
This is currently specced. The larger problem would be what to do when multiple of such operations are overlapping..... We'll have to see how it works out in performance tests.
I would opt to have the sender check it and reduce the number of messages sent otherwise it would risk sending data that won't be accepted. The protocol itself wouldn't be harmed since it would pick up where left in the next sync cycle.